webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path'); // eslint-disable-line no-unused-vars
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: `${__dirname}/dist`,
+    filename: 'bundle.js',
```

Comment:
The instructions stated that file should be named `main.js` [See this screenshot from odin](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png)

Reply:
Oh, sorry about that, thanks for your reply.